core/modules/qana/testPlugins.cc

```diff
@@ -123,24 +123,6 @@ bool columnRefPtrComparisonPredicate(query::ColumnRef::Ptr lhs, query::ColumnRef
 }
 
 
-BOOST_DATA_TEST_CASE(OrderBy, QUERIES, query) {
-    auto parser = parser::SelectParser::newInstance(query.query);
-    try {
-        parser->setup();
-    } catch(parser::ParseException const& e) {
-        BOOST_TEST_MESSAGE("parse exception: " << e.what());
-    }
-    auto selectStatement = parser->getSelectStmt();
-    auto validOrderByColumns = qana::PostPlugin::getValidOrderByColumns(*selectStatement);
```

Comment:
PostPlugin::getValidOrderByColumns was removed, the ORDER BY column verification was moved to TablePlugin and is incorporated into any unit test that uses the TablePlug in (the plugin will fail if it can not match the ORDER BY columns to SELECT list columns.